v3/setup/capcom.sh

```diff
@@ -7,26 +7,24 @@ if [ "${NODE_ROLE}" != "control" ]  && [ "${NODE_ROLE}" != "it-hybrid" ]; then
     exit 0
 fi
 
-if [ "$(etcdctl get capcom-bootstrapped)" == "true" ]; then
+if [ "$(etcdctl get /capcom/config/bootstrapped)" == "true" ]; then
```

Comment:
I think that would be cleaner - everything in `config` is an actual config value. Not sure  I love the word cluster-state but something along those lines.